locales/en-US/app.ftl

```diff
@@ -20,6 +20,32 @@ AppHeader--app-header = <header>{ -profiler-brand-name }</header> — <subheader
 AppHeader--github-icon =
     .title = Go to our git repository (this opens in a new window)
 
+## AppViewRouter
+
+AppViewRouter--error-message-from-addon =
+    .message = Couldn’t retrieve the profile from { -firefox-brand-name }.
+
+AppViewRouter--error-message-unpublished =
+    .message = Couldn’t retrieve the profile from { -firefox-brand-name }.
+
+AppViewRouter--error-message-from-file =
+    .message = Couldn’t read the file or parse the profile in it.
+
+AppViewRouter--error-message-local =
+    .message = Not implemented yet.
+
+AppViewRouter--error-message-public =
+    .message = Could not download the profile.
+
+AppViewRouter--error-message-from-url =
+    .message = Could not download the profile.
+
+AppViewRouter--error-message-compare =
+    .message = Could not retrieve the profile
+
+AppViewRouter--home-special-message =
+    .specialMessage = The URL you came in on was not recognized.
```

Comment:
Doesn't this sound a bit weird? Assuming I understand the original intent, maybe `The URL you tried to reach was not recognized.`, or something similar?

Reply:
Yes, you are right, I have changed the string. Let's see what @julienw says about this.